classes/user/PKPUserDAO.inc.php

```diff
@@ -243,7 +244,7 @@ function getAllReviewers($contextId) {
 	 * @param $reviewRoundId int Also filter users assigned to this round of the given submission
 	 * @return array Users
 	 */
-	function getFilteredReviewers($contextId, $stageId, $doneMin = null, $doneMax = null, $avgMin = null, $avgMax = null, $lastMin = null, $lastMax = null, $activeMin = null, $activeMax = null, $interests = array(), $submissionId = null, $reviewRoundId = null) {
+	function getFilteredReviewers($contextId, $stageId, $name = null, $doneEnabled = null, $doneMin = null, $doneMax = null, $avgEnabled = null, $avgMin = null, $avgMax = null, $lastEnabled = null, $lastMin = null, $lastMax = null, $activeEnabled = null, $activeMin = null, $activeMax = null, $interests = array(), $submissionId = null, $reviewRoundId = null) {
```

Comment:
Yes, the second approach would make it somewhat better.

I've mused about the possibility of creating and passing in some kind of object to describe filtering, but let's not do that just yet.